engine/schema/src/main/resources/META-INF/db/schema-41610to41700.sql

```diff
@@ -638,4 +638,53 @@ CREATE VIEW `cloud`.`domain_router_view` AS
             left join
         `cloud`.`async_job` ON async_job.instance_id = vm_instance.id
             and async_job.instance_type = 'DomainRouter'
-            and async_job.job_status = 0;
\ No newline at end of file
+            and async_job.job_status = 0;
+
+----- PR Quota custom tariffs #0000 -----
+-- Create column 'uuid'
+ALTER TABLE cloud_usage.quota_tariff
+ADD COLUMN  `uuid` varchar(40);
+
+UPDATE  cloud_usage.quota_tariff
+SET     uuid = UUID()
```

Comment:
not sure, but this might lead to multiple records with the same ID, i.e. uuid() would be evaluated before the update execution is done.

Reply:
@DaanHoogland, it generates an UUID to each record updated (quite similar, but different):

```sql
MariaDB [cloud_usage]> SELECT id, usage_name, uuid FROM cloud_usage.quota_tariff;
+----+------------------------+--------------------------------------+
| id | usage_name             | uuid                                 |
+----+------------------------+--------------------------------------+
|  1 | RUNNING_VM             | 81c74b75-7ac5-11ec-9c0a-5254006051f5 |
|  2 | ALLOCATED_VM           | 81c750e0-7ac5-11ec-9c0a-5254006051f5 |
|  3 | IP_ADDRESS             | 81c75207-7ac5-11ec-9c0a-5254006051f5 |
|  4 | NETWORK_BYTES_SENT     | 81c752e0-7ac5-11ec-9c0a-5254006051f5 |
|  5 | NETWORK_BYTES_RECEIVED | 81c753b2-7ac5-11ec-9c0a-5254006051f5 |
|  6 | VOLUME                 | 81c75467-7ac5-11ec-9c0a-5254006051f5 |
|  7 | TEMPLATE               | 81c75516-7ac5-11ec-9c0a-5254006051f5 |
|  8 | ISO                    | 81c755dc-7ac5-11ec-9c0a-5254006051f5 |
|  9 | SNAPSHOT               | 81c7568f-7ac5-11ec-9c0a-5254006051f5 |
| 10 | SECURITY_GROUP         | 81c75748-7ac5-11ec-9c0a-5254006051f5 |
| 11 | LOAD_BALANCER_POLICY   | 81c757f1-7ac5-11ec-9c0a-5254006051f5 |
| 12 | PORT_FORWARDING_RULE   | 81c758a1-7ac5-11ec-9c0a-5254006051f5 |
| 13 | NETWORK_OFFERING       | 81c7595e-7ac5-11ec-9c0a-5254006051f5 |
| 14 | VPN_USERS              | 81c75a12-7ac5-11ec-9c0a-5254006051f5 |
| 15 | CPU_SPEED              | 81c75acd-7ac5-11ec-9c0a-5254006051f5 |
| 16 | vCPU                   | 81c75b7a-7ac5-11ec-9c0a-5254006051f5 |
| 17 | MEMORY                 | 81c75e5d-7ac5-11ec-9c0a-5254006051f5 |
| 18 | VM_DISK_IO_READ        | 81c75fbd-7ac5-11ec-9c0a-5254006051f5 |
| 19 | VM_DISK_IO_WRITE       | 81c76090-7ac5-11ec-9c0a-5254006051f5 |
| 20 | VM_DISK_BYTES_READ     | 81c7614b-7ac5-11ec-9c0a-5254006051f5 |
| 21 | VM_DISK_BYTES_WRITE    | 81c76203-7ac5-11ec-9c0a-5254006051f5 |
| 22 | VM_SNAPSHOT            | 81c762c2-7ac5-11ec-9c0a-5254006051f5 |
```